.github/workflows/CI.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         manylinux: auto
         command: build
         rust-toolchain: stable
-        args: --release --sdist -o dist --find-interpreter
+        args: --release --sdist -o dist --interpreter python3.9 --interpreter python3.10 --interpreter python3.11
```

Comment:
Unrelated, but Python 3.12 is not supported. Previously, we were building for:
```
Found CPython 3.9 at /opt/python/cp39-cp39/bin/python, CPython 3.10 at /opt/python/cp310-cp310/bin/python, CPython 3.11 at /opt/python/cp311-cp311/bin/python, CPython 3.12 at /opt/python/cp312-cp312/bin/python, PyPy 3.10 at /opt/python/pp310-pypy310_pp73/bin/python
```
Now the last one (PyPy 3.10) does not seem to be detected. Not sure whether it is important?

Reply:
I'm not sure I understand... 3.12 was supported all along. There are even wheels for it, e.g. [this one](https://pypi.org/project/starlark-pyo3/#starlark_pyo3-2024.1.4-cp312-none-win_amd64.whl). Nonetheless, your hackery with Python versions prompted me to get support for 3.13 squared away, see #25. Once that's in, this PR will have some conflicts (and should no longer need hackery of the build config).

Reply:
Updated this for you based on recent changes.

Reply:
That's a mistake, I forgot 3.12 in my modification. The goal was to avoid 3.13 which was unsupported at that time. Anyway, since you fixed it I will rebase and remove my changes. :)